02/banka/banka.py

```diff
@@ -23,3 +23,50 @@
 #
 # Pro tento pokročilý způsob je však třeba použít pokročilou knihovnu pro práci
 # s parametry příkazové řádky, jako např. argparse nebo click.
+
+import argparse
+
+def amount_on_account(account_num):
+    try:
+        with open(account_num) as file:
+            return int(file.read())
+    except FileNotFoundError:
+        print("The sender's account doesn't exist.")
+        exit()
+
+    
+
+# Create parser
+parser = argparse.ArgumentParser(description="bank transfer")
+
+# Adding arguments
+parser.add_argument('--From', '--from', type=str, help="senders_account")
+parser.add_argument('--to', type=str, help="income_account")
+parser.add_argument('--amount', type=int, help="amount")
+
+
+args = parser.parse_args()
+
+if args.From and args.to and args.amount:
+
+    # Finding the amounts on the respective accounts
+    senders_amount = amount_on_account(args.From)
+    income_amount = amount_on_account(args.to)
```

Comment:
Označení proměnné mi připadá zavádějící. Myslím si, že income_amount si jde snadno splést s výší posílané částky.
Prosím přejmenovat například na:
```suggestion
    recipient_amount = amount_on_account(args.to)
```

Reply:
Díky, opraveno